resources/default_pipeline_environment.yml

```diff
@@ -211,10 +211,10 @@ steps:
       - 'deployDescriptor'
       - 'pipelineConfigAndTests'
     cf_native:
-      dockerImage: 'ppiper/cf-cli'
+      dockerImage: 'ppiper/cf-cli:v5'
```

Comment:
Why is this change needed? The latest version i.e v5 of ppiper/cf-cli docker image is pulled from the docker hub right?

Reply:
We need to fix the version temporary, because the next latest version will be cf version 7.2 than may have incompatible changes. To prevent this, we fix the version temporary.